pkg/conversion/conversion.go

```diff
@@ -0,0 +1,124 @@
+package conversion
```

Comment:
Would it be better to have version conversions move to their respective directories ? Wouldn't this file grow too long when we add more versions ?